install/gcp/bootstrap/gcp_envoy_bootstrap.json

```diff
@@ -76,4 +76,27 @@
       }
     }
   }
+  {{ if .stackdriver }}
+  ,
+  "tracing": {
+    "http": {
+      "name": "envoy.tracers.opencensus",
+      "config": {
+      "stackdriver_exporter_enabled": true,
+      "stackdriver_project_id": "{{ .stackdriverProjectID }}",
+      "stdout_exporter_enabled": {{ .stackdriverDebug }},
+      "incoming_trace_context": ["CLOUD_TRACE_CONTEXT", "TRACE_CONTEXT", "GRPC_TRACE_BIN"],
+      "outgoing_trace_context": ["CLOUD_TRACE_CONTEXT", "TRACE_CONTEXT", "GRPC_TRACE_BIN"],
+      "trace_config":{
+        "constant_sampler":{
+          "decision": "ALWAYS_PARENT"
+        },
+        "max_number_of_annotations": {{ .stackdriverMaxAnnotations }},
+        "max_number_of_attributes": {{ .stackdriverMaxAttributes }},
+        "max_number_of_message_events": {{ .stackdriverMaxEvents }},
+        "max_number_of_links": 200,
```

Comment:
@nareddyt Can you take a quick look at the defaults here (all templated stuff is set to 200)? Is this reasonable to run in production?

Reply:
Ok, looks reasonable.

Reply:
Yes, I think that is fine. We do the same in our code, setting them to the max number stackdriver allows. We did not see much degradation in performance.